### PR TITLE
[sailfish-browser] Add maxRowCount to overlay and renaming toolsHeight to rowHeight. Contributes to JB#51715

### PR DESCRIPTION
**Pull request by atatarov**

### CODE REVIEW

**Review by okodron — APPROVED** [review]

**Review by paveltkv — APPROVED** [review]

**Comment by rainemak** [comment]
This causes odd looking flickering and change to the host that is visible in the toolbar all the time.

Reason for that is (as far as my eye can spot if) when opening secondary layer it's now animating from to bottom... not from from the top of the first row.

**Review by rainemak — CHANGES_REQUESTED** [review]
Let's fix that flickering before merging.

**Review by rainemak — CHANGES_REQUESTED** [review]
Almost there...

Please change CaptivePortalToolBar.qml as well...

```
    readonly property real rowHeight: height
    readonly property int maxRowCount: 1
```
There we have higher row => let's keep to API in sync that OverlayAnimator keeps on working.

File: src/pages/components/CaptivePortalToolBar.qml [hunk 1]
```diff
@@ -16,7 +16,7 @@ Column {
     id: toolBarRow
 
     property string url
-    readonly property real toolsHeight: height
+    readonly property real rowHeight: height
```
Comment (rainemak) [inline src/pages/components/CaptivePortalToolBar.qml:19]:
```
readonly property int maxRowCount: 1
```
This is still missing... Please also rebase.

**Review by rainemak — COMMENTED** [review]
More comments.

**Review by rainemak — APPROVED** [review]
:+1:

**Pull request merged**